pkg/comp-functions/functions/vshnmariadb/user_management.go

```diff
@@ -61,7 +61,8 @@ func addUser(comp common.Composite, svc *runtime.ServiceRuntime, username string
 				"crossplane.io/external-name": username,
 			},
 			Labels: map[string]string{
-				runtime.ProviderConfigIgnoreLabel: "true",
+				runtime.ProviderConfigIgnoreLabel:      "true",
+				"appcat.vshn.io/webhook-allowdeletion": "true",
```

Comment:
Having the name of this label as a const in the `runtime` package would be nice.

Reply:
Good point. I've added a new const and replaced all labels